less/lists.less

```diff
@@ -381,14 +381,22 @@
 
 .section.associations, .view-details-associations, .list {
   .list-item {
-    flex-basis: 48%;
-    @media (min-width: @large-width){
-      flex-basis: 32%;
+    @media (min-width: 2400px) {
+      flex-basis: 16% !important;
```

Comment:
Why are you using `!important`? It works as well without. We should avoid using this property if possible.

I think you did the same in your previous PR. This could be corrected too.